src/Core/Application.php

```diff
@@ -1280,7 +1280,8 @@ public function manage(string $kernel, $request)
         $kernel = $this->make($kernel);
 
         $response = $kernel->handle($request);
-        $response->send();
+        $response->sendHeaders();
```

Comment:
@ogorzalka Separating the calls makes the `shutdown` hook working ? It's weird as the `send` method only calls those two methods behind the scene.